### PR TITLE
autotune: support decimation of autotune partition data

### PR DESCRIPTION
**Pull request by mlyle**
Needed to save ram on RE1.  Unfortunately slightly cripples new-autotune on RE1 but 1.25ms resolution isn't completely awful.

### CODE REVIEW

File: flight/Modules/Autotune/autotune.c [hunk 1]
```diff
@@ -361,7 +367,9 @@ static void AutotuneTask(void *parameters)
 		PIOS_Thread_Sleep(25);
 	}
 
-	uint16_t buf_size = sizeof(*at_averages) * ident_wiggle_points;
+	decim_wiggle_points /= AUTOTUNE_AVERAGING_DECIMATION;
```
Comment (tracernz) [inline flight/Modules/Autotune/autotune.c]: I don't see this being initialized anywhere? Is it supposed to be `decim_wiggle_points = ident_wiggle_points / AUTOTUNE_AVERAGING_DECIMATION`?

Comment (mlyle) [reply flight/Modules/Autotune/autotune.c]: :D yes

Comment (tracernz) [reply flight/Modules/Autotune/autotune.c]: :wink:

**Comment by mlyle** [comment]
Woops I had forgotten to pus this :P

File: flight/Modules/Autotune/autotune.c [hunk 2]
```diff
@@ -1,4 +1,4 @@
-/**
+
```
Comment (tracernz) [inline flight/Modules/Autotune/autotune.c]: yudelete? :wink:

**Comment by mlyle** [comment]
Flew #1618, #1610, #1599... plus a change to decimate = 2 on Lux.  Set to 88Hz second order filter.  Did an autotune.  

Requires disabling the lowpass filter in the autotune script (for filtering change), but the results are good:

bokeh_plot 25
 
http://dronin.org/tunes#?key=ahFzfmRyb25pbi1hdXRvdG93bnIYCxILVHVuZVJlc3VsdHMYgICA0OCU3AgM

**Comment by mlyle** [comment]
@mluessi any comment before this merges?  It flew OK on Lux.

**Review by tracernz — APPROVED** [review]
This looks good to me, and it's proven to work okay.

**Pull request merged**